fabric-biome-api-v1/src/main/java/net/fabricmc/fabric/mixin/biome/MultiNoiseUtilMultiNoiseSamplerMixin.java

```diff
@@ -28,7 +28,7 @@
 import net.fabricmc.fabric.impl.biome.MultiNoiseSamplerHooks;
 
 @Mixin(MultiNoiseUtil.MultiNoiseSampler.class)
-public class MixinMultiNoiseSampler implements MultiNoiseSamplerHooks {
+public class MultiNoiseUtilMultiNoiseSamplerMixin implements MultiNoiseSamplerHooks {
```

Comment:
For cases like these I think the task should allow to just use the inner name too, but doesn't really matter here